bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/climatecontrol/ClimateControlHandler.java

```diff
@@ -0,0 +1,83 @@
+package org.openhab.binding.boschshc.internal.devices.climatecontrol;
```

Comment:
After I read to all the new content and guide lines we shall follow to a merge, I suggest to start here soon with class and author comments.
Compare with [GuideLines Documentation](https://www.openhab.org/docs/developer/guidelines.html#c-documentation)

I need to do the same for my merged PR and will try to add many more documentaton to missing classes.